.devops/deploy-pipelines.yml

```
@@ -138,3 +138,11 @@ stages:
               storage: '$(PRODUCTION_STORAGE_ACCOUNT_NAME)'
               containerName: '$web'
             displayName: Deploy to container blob
+
+          - task: AzureCLI@1
+            displayName: 'Purge CDN'
+            inputs:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              scriptLocation: inlineScript
+              inlineScript: |
+                call az cdn endpoint purge -g $(RESOURCE_GROUP_AZURE) -n $(ENDPOINT_AZURE) --profile-name $(PROFILE_NAME_CDN_AZURE) --content-paths "/*"
```

We can rename variables to be the same we use in other pipelines:
```suggestion
	                call az cdn endpoint purge -g $(PRODUCTION_RESOURCE_GROUP_NAME) -n $(PRODUCTION_CDN_ENDPOINT) --profile-name $(PRODUCTION_CDN_PROFILE_NAME) --content-paths "/*"
```

Do they make sense?

---

ok 👍